client/src/auth/auth0-provider-with-history.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { Auth0Provider } from "@auth0/auth0-react";
+
+const Auth0ProviderWithHistory = ({ children }) => {
+  const domain = process.env.REACT_APP_AUTH0_DOMAIN;
+  const clientId = process.env.REACT_APP_AUTH0_CLIENT_ID;
+
+
+
+  return (
+    <Auth0Provider
+      domain={domain}
+      clientId={clientId}
+      redirectUri={`${window.location.origin}/feed`}
+    >
+      {children}
+    </Auth0Provider>
+  );
+};
+
+export default Auth0ProviderWithHistory;
```

Comment:
Get rid of this component. Instead, wrap using `<Auth0Provider>` directly. Also add newline